saleor/settings.py

```diff
@@ -385,3 +377,27 @@ def get_host():
 }
 
 SITE_SETTINGS_ID = 1
+
+AUTHENTICATION_BACKENDS = [
+    'saleor.registration.backends.facebook.CustomFacebookOAuth2',
+    'saleor.registration.backends.google.CustomGoogleOAuth2',
+    'django.contrib.auth.backends.ModelBackend',
+]
+
+SOCIAL_AUTH_PIPELINE = [
+    'social_core.pipeline.social_auth.social_details',
+    'social_core.pipeline.social_auth.social_uid',
+    'social_core.pipeline.social_auth.auth_allowed',
+    'social_core.pipeline.social_auth.social_user',
+    'social_core.pipeline.social_auth.associate_by_email',
+    'social_core.pipeline.user.create_user',
+    'social_core.pipeline.social_auth.associate_user',
+    'social_core.pipeline.social_auth.load_extra_data',
+    'social_core.pipeline.user.user_details',
+]
+
+SOCIAL_AUTH_USERNAME_IS_FULL_EMAIL = True
+SOCIAL_ATUTH_USER_MODEL = AUTH_USER_MODEL
```

Comment:
Yes, it works because if it doesn't find `SOCIAL_AUTH_USER_MODEL` then it will use `AUTH_USER_MODEL`. Maybe I just delete this variable?